ocs_ci/ocs/provider_client_upgrade.py

```
@@ -193,6 +193,9 @@ class ProviderClusterOperatorUpgrade(ProviderUpgrade):
 
     """
 
+    def __init__(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)